keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/SubmittedDeliveryConfig.kt

```diff
@@ -12,8 +12,8 @@ const val DEFAULT_SERVICE_ACCOUNT = "keel@spinnaker.io"
 
 @Description("A manifest specifying the environments and resources that comprise an application.")
 data class SubmittedDeliveryConfig(
-  val name: String,
   val application: String,
+  val name: String = "$application-manifest",
```

Comment:
Default the name here (had to change property order as otherwise I can't use `application` in the definition of `name`)